files/dashboards/fluentd/openshift-logging-dashboard.json

```
@@ -1400,7 +1400,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "100 - max by(plugin_id)(fluentd_output_status_buffer_available_space_ratio)",
+              "expr": "100 - min by(plugin_id)(fluentd_output_status_buffer_available_space_ratio)",
```

I'm interpreting the title of the graph as "it should show the available buffer percentage". If that's the case in addition to the change from `max` to `min` we should also remove the `100 - `:

```suggestion
              "expr": "min by(plugin_id)(fluentd_output_status_buffer_available_space_ratio)",
```

If we want to keep showing the "usage" like we currently do, I'd suggest we change the name of the graph accordingly (`FluentD Buffer Usage`, for example). The change from `max` to `min` is still valid in that case.

---

I think "FluentD Buffer Usage" is better choice, because usage is intuitive than availability usually for users.